db/seeds.rb

```diff
@@ -56,7 +56,7 @@
     incident = build :incident, incident_attrs
     # Validations don't pass for incomplete incidents. They do in real life,
     # but they don't because we create objects in reverse order here.
-    incident.save validate: false
+    incident.save validate: incident_type != :incomplete
```

Comment:
Not strictly related to this change. I just realized that I should've been doing this the whole time.